libs/product/src/selectors/configurable-product/configurable-product.selectors.unit.spec.ts

```diff
@@ -376,7 +376,28 @@ describe('Configurable Product Selectors | unit tests', () => {
 				a: {
 					color: ['0', '1', '2'],
 					size: ['0', '1', '2'],
-					material: ['0', '1', '2']
+					material: ['0', '2', '1']
+				}
+			});
+
+			expect(selector).toBeObservable(expected);
+		});
+
+		it('returns expected dictionary when variants are out of stock', () => {
```

Comment:
Does this test the case where there are no applied attributes?

Reply:
yes